Framework/Algorithms/src/SofQWCentre.cpp

```diff
@@ -327,8 +327,8 @@ void SofQWCentre::makeDistribution(API::MatrixWorkspace_sptr outputWS,
 
   const size_t numQBins = outputWS->getNumberHistograms();
   for (size_t i = 0; i < numQBins; ++i) {
-    MantidVec &Y = outputWS->dataY(i);
-    MantidVec &E = outputWS->dataE(i);
+    auto &Y = outputWS->mutableY(i);
+    auto &E = outputWS->mutableE(i);
     std::transform(Y.begin(), Y.end(), Y.begin(),
                    std::bind2nd(std::divides<double>(), widths[i + 1]));
     std::transform(E.begin(), E.end(), E.begin(),
```

Comment:
I think this method is the same as `WorkspaceHelpers::makeDistribution` from `MantidAPI/WorkspaceOpOverloads.h`, i.e., can be removed completely?